### ISSUE TITLE
test_get_country_by_ip fails in some circumstances

### ISSUE THREAD

**Issue by zadigus** [saleor/saleor#4868]
### What I'm trying to achieve

I am trying to make the test `test_get_country_by_ip` run successfully in all configurations.

### Steps to reproduce the problem

1. `pytest -ra`

Under some configurations, the `test_get_country_by_ip` fails with the following error:
```
[gw0] linux -- Python 3.6.9 /var/jenkins_home/workspace/backend-unit/venv/bin/python

ip_data = {'country': {'iso_code': 'UNKNOWN'}}, expected_country = None
monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x7f42c0557e10>

    @pytest.mark.parametrize(
        "ip_data, expected_country",
        [
            ({"country": {"iso_code": "PL"}}, Country("PL")),
            ({"country": {"iso_code": "UNKNOWN"}}, None),
            (None, None),
            ({}, None),
            ({"country": {}}, None),
        ],
    )
    def test_get_country_by_ip(ip_data, expected_country, monkeypatch):
>       monkeypatch.setattr("saleor.core.utils.georeader.get", Mock(return_value=ip_data))
E       AttributeError: 'Reader' object attribute 'get' is read-only
```

### What I expected to happen

All tests are successful.

### Screenshots

**System information**
Operating system: Ubuntu 18.04

I tried with python 3.6 and 3.7, on my jenkins server and my laptop. On the Jenkins server, it always fails, on the laptop it's always fine. Until now, I have tried to run the tests on
* docker image `nikolaik/python-nodejs:python3.6-nodejs12` --> test fails
* docker image  `nikolaik/python-nodejs:python3.7-nodejs12` --> test fails
* my laptop with python 3.6.8 --> test successful
* my laptop with python 3.7.3 --> test successful

Have you ever had that error? It's for sure a configuration issue, I can't figure out what.

**Comment by beatboxchad** [comment]
Yep! Me too!

**Comment by tamassoltesz** [comment]
Yep! Me too! I am currently trying to investigate what's happening.

**Comment by wrvdklooster** [comment]
@tamassoltesz Did you find the cause? I am having the same issue now. We forked Saleor 2.10.1 and while moving from Python 3.6 tp 3.8 this test fails.

**Comment by maarcingebala** [comment]
Geolocalization and the mentioned test is no longer available in Saleor 3.0/3.1.
